pepys_import/file/highlighter/support/line.py

```diff
@@ -96,7 +96,10 @@ def record(self, tool: str, field: str, value: str, units: str = "n/a"):
         self.highlighted_file.fill_char_array_if_needed()
 
         tool_field = tool + "/" + field

```

Comment:
@robintw - since a default value is provided for `units`, I guess that in the if structure below  `units` will never have a value of `None`.

(but that may be a shortcoming in my Python skills).

Reply:
Good catch, that default value was meant to be removed - I'll push a new commit

Reply:
Fixed in 
91eb3a8